Batch.roboFontExt/lib/batch.py

```diff
@@ -134,6 +138,7 @@ def build(self, sources=[]):
             size="auto",
             controller=self,
         )
+        self.w.setDefaultButton(self.w.getItem("generate"))
```

Comment:
this can also be set as entry in the EZWindow constructor: `defaultButton="generate"